shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/java/org/apache/shardingsphere/sql/parser/integrate/asserts/segment/table/TableFactorAssert.java

```diff
@@ -43,8 +43,10 @@
      * @param expected expected TableFactor
      */
     public static void assertIs(final SQLCaseAssertContext assertContext, final TableFactorSegment actual, final ExpectedTableFactor expected) {
-        if (null != actual.getTable() && actual.getTable() instanceof SimpleTableSegment) {
-            TableAssert.assertIs(assertContext, (SimpleTableSegment) actual.getTable(), expected.getTable());
+        if (null != actual.getTable()) {
+            if (actual.getTable() instanceof SimpleTableSegment) {
+                TableAssert.assertIs(assertContext, (SimpleTableSegment) actual.getTable(), expected.getTable());
+            }
```

Comment:
Is there any difference?

Reply:
This change is for future subquery testing，if table is subqueryTable.